### ISSUE TITLE
Project Review

### ISSUE THREAD

**Issue by CPernet** [Brainhack-Proceedings-2015/Craddock-AMX-Centrality#1]
Nice implementation of connectivity measures with increase speed + validation.
I have no concerns given that you obtain the same results in the validation phase. I a little surprised that you get 'identical' values for DC with sparsity threshold given that you use histograms. First can you confirm the values are identical? Also how do you justify the number of bins? typically the number of bin in an histogram is 'optimized' based on the length of the data see eg https://estatistics.eu/what-is-statistics-graph-figures-histogram/ 

few typos - see pdf upload[Craddock-AMX-Centrality-Reviwed.pdf](https://files.zenhub.io/56d74d197137874e451ccfe5)
Uploaded using [**ZenHub.io**](http://www.zenhub.io)

**Comment by nicholsn** [comment]
@ccraddock can you respond to:

> First can you confirm the values are identical? 
> Also how do you justify the number of bins?

Have you had a chance to incorporate:

> few typos - see pdf [uploadCraddock-AMX-Centrality-Reviwed.pdf](https://files.zenhub.io/56d74d197137874e451ccfe5)

**Comment by ccraddock** [comment]
We would like to begin by thinking Dr. Pernet for his through and insightful review.

We verified that the results were the same using the spatial concordance correlation coefficient (CCC) between centrality maps. Unlike Pearson's correlation, which is not sensitive to differences in shifts and variance, CCC measures how identical the spatial patterns are. The results for all pairs of images were > 0.99 with many equal to 1. We agree though that the term "identical" is a little extreme, so we changed it to "high similar".

It is true that the binwidth has a huge impact on the quality of kernel density estimation using histograms and when there isn't a clear reason to choose a particular binwidth, it can be chosen using cross validation. In this case, the bindwidth determines the range of values that are considered ties, which also impacts the tendency of the algorithm to return more values than requested, at a cost of the speed that the algorithm converges to the sparsity threshold. We chose an effective bindwidth of 1/(50*100) which in our experience offers a good trade off between these extremes. We have updated the text to describe this in more detail.

Thanks again for helping to improve this project report.

Regards,
Cameron
